sktime/annotation/online_anomaly/online_bayes_cpd.py

```diff
@@ -0,0 +1,289 @@
+"""Module for online anomaly detection using Bayesian methods.
+
+References
+----------
+TODO
+
+"""
+
+import numpy as np
+import pandas as pd
+from scipy import stats
+
+from sktime.annotation.base import BaseSeriesAnnotator
+
+
+class StudentTDistribution:
```

Comment:
the distribution exists already in `sktime.proba.t`, no?

On first glance, I would have modelled this as a distribution fitter, inheriting from the "parameter estimator" template in `param_est`. However, it does not fit entirely the interface, so we can leave it as is for now.